pages/pipelines/permissions.md.erb

```diff
@@ -50,16 +51,32 @@ if [[ ":$BUILDKITE_BUILD_CREATOR_TEAMS:" != *":ops:"* ]]; then
 fi
 ```
 
-## Frequently Asked Questions
+### Frequently Asked Questions
 
-### Will users (and API tokens) still have access to their pipelines?
+#### Will users (and API tokens) still have access to their pipelines?
 When you enable Teams we’ll create a default team called “Everyone”, containing all your users and pipelines. This ensures that users, and their API tokens, will still have access to their pipelines.
 
-### How does Teams work with SSO?
+#### How does Teams work with SSO?
 When a user joins the organization via SSO, they’ll be automatically added to any teams that have the “Automatically add new users to this team” setting enabled.
 
-### Can I delete the “Everyone” team?
+#### Can I delete the “Everyone” team?
 Yes you can delete or edit the “Everyone” team. To ensure uninterrupted access to pipelines we recommend creating new teams before deleting the “Everyone” team.
 
-### Once enabled, can I disable Teams?
+#### Once enabled, can I disable Teams?
 You can disable teams by deleting all your teams, and then selecting “Disable Teams”.
+
+## Member Permissions
```

Comment:
```suggestion
## Member Permissions without teams
```

Reply:
I'm not sure about this one, because in the first paragraph I've said "These permissions can be used both with or without Teams enabled." 😅